grocy/rootfs/var/www/grocy/localization/nl/component_translations.po

```diff
@@ -0,0 +1,47 @@
+# Translators:
```

Comment:
Maybe add a comment here to remove on the next version number?

Reply:
:)